### PR TITLE
Meta tx fixes

### PR DESCRIPTION
**Pull request by nick**
- Attestation valid on either proxy or owner. 
- GraphQL server config override for proxies.

### CODE REVIEW

File: packages/graphql/src/server.js [hunk 1]
```diff
@@ -15,7 +15,8 @@ try {
 
 setNetwork(process.env.NETWORK || 'test', {
   performanceMode: false,
-  useMetricsProvider: process.env.USE_METRICS_PROVIDER === 'true'
+  useMetricsProvider: process.env.USE_METRICS_PROVIDER === 'true',
+  proxyAccountsEnabled: process.env.PROXY_ACCOUNTS_ENABLED === 'true'
```
Comment (franckc) [inline packages/graphql/src/server.js]: I don't see that used anywhere so I assume it is for future ?

File: packages/graphql/src/resolvers/IdentityEvents.js [hunk 2]
```diff
@@ -79,9 +80,15 @@ export function identity({ id, ipfsHash }) {
...
+    let accounts = id
     if (!ipfsHash) {
+      const owner = await proxyOwner(id)
+      if (owner) {
+        accounts = [id, owner]
+      }
+
       const events = await contracts.identityEvents.eventCache.getEvents({
-        account: id
+        account: accounts
```
Comment (franckc) [inline packages/graphql/src/resolvers/IdentityEvents.js]: I haven't paid close attention to the EventCache interface. I assume it expects addresses checksummed and both id and owner are indeed checksummed ?

File: packages/graphql/src/resolvers/IdentityEvents.js [hunk 3]
```diff
@@ -114,7 +121,7 @@ export function identity({ id, ipfsHash }) {
         'avatarUrl',
         'description'
       ]),
-      ...getAttestations(id, data.attestations || []),
+      ...getAttestations(accounts, data.attestations || []),
```
Comment (franckc) [inline packages/graphql/src/resolvers/IdentityEvents.js]: Does getAttestation need to be updated to de-dupe attestations that are identical between proxy and owner ?

File: packages/graphql/src/server.js (hunk 1 above)
Comment (nick) [reply packages/graphql/src/server.js]: Right now proxies are enabled with a localStorage var but we need the GraphQL server to always have it on for performance mode

File: packages/graphql/src/resolvers/IdentityEvents.js (hunk 2 above)
Comment (nick) [reply packages/graphql/src/resolvers/IdentityEvents.js]: Yep they are

File: packages/graphql/src/resolvers/IdentityEvents.js (hunk 3 above)
Comment (nick) [reply packages/graphql/src/resolvers/IdentityEvents.js]: It should only look at the latest IPFS hash which shouldn't contain any duplicates

File: packages/graphql/src/server.js (hunk 1 above)
Comment (franckc) [reply packages/graphql/src/server.js]: Got it. Sg.

File: packages/graphql/src/resolvers/IdentityEvents.js (hunk 3 above)
Comment (franckc) [reply packages/graphql/src/resolvers/IdentityEvents.js]: ah right. perfect.

File: packages/graphql/src/resolvers/IdentityEvents.js (hunk 2 above)
Comment (franckc) [reply packages/graphql/src/resolvers/IdentityEvents.js]: sweet

**Pull request merged**